src/components/pages/datasets/kved/KvedList.jsx

```diff
@@ -17,7 +16,7 @@ const KvedList = () => {
     {
       header: t('class'),
       defaultSelected: true,
-      prop: 'name',
+      prop: i18n.language === 'uk' ? 'name' : 'name_en',
```

Comment:
use `getLocaleField`

Reply:
It's done here: https://github.com/3v-workspace/DataOcean/pull/195/files